feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -117,6 +130,159 @@ export class ConfiguratorAttributeNumericInputFieldComponent
     super.ngOnDestroy();
   }
 
+  /**
+   * Returns a concatenated help text for multiple intervals.
+   */
+  getHelpTextForInterval(): string {
```

Comment:
This method must take i18n into account. It always creates the text in en locale (which is what the engine sends). See that from a test in German
![Screen Shot 2022-03-30 at 09 31 18](https://user-images.githubusercontent.com/33626130/160776717-150066f3-c221-4f85-844d-ca48d7831f59.png)

Reply:
Done